metecho/api/hook_serializers.py

```diff
@@ -32,9 +32,10 @@ class PrBranchSerializer(serializers.Serializer):
 
 
 class PrSerializer(serializers.Serializer):
-    merged = serializers.BooleanField()
+    merged = serializers.BooleanField(required=False)
```

Comment:
@wlonk Apparently `pull_request_review` events have `merged_at`, but no `merged` key. This works, but feel free to handle it differently if you like.